ansible_collections/arista/cvp/plugins/modules/cv_device_v3.py

```diff
@@ -186,6 +186,9 @@ def main():
 
     if ansible_module.params['state'] == 'absent':
         ansible_module.fail_json(msg='State==absent is not yet supported !')
+    # Check if the device list is empty
+    if len(ansible_module.params['devices']) == 0:
+        ansible_module.fail_json(msg='Aborting, "devices" list is empty')
```

Comment:
@titom73 Thinking more about this. Do we want to abort with `fail` or just return with no actions?
In the beginning of a new fabric buildout, the devices list might be empty (all switches have `is_deployed` set to false), but you still want the push configlets and create containers.

Reply:
The original issue came from `null`/`None` value, so we should really test for that and not an empty list.